FV3/atmos_cubed_sphere/tools/fv_diagnostics.F90

```diff
@@ -1077,6 +1077,23 @@ subroutine fv_diag_init(Atm, axes, Time, npx, npy, npz, p_ref)
         Atm(n)%nudge_diag%nudge_v_dt(isc:iec,jsc:jec,npz) = 0.0
     endif
 
+    idiag%id_t_dt_phys = register_diag_field('dynamics', &
+          't_dt_phys', axes(1:3), Time, &
+          'temperature tendency from physics', 'K/s', &
+          missing_value=missing_value)
+    if (idiag%id_t_dt_phys > 0) then
+         allocate(Atm(n)%physics_tendency_diag%t_dt(isc:iec,jsc:jec,npz))
+         Atm(n)%physics_tendency_diag%t_dt(isc:iec,jsc:jec,1:npz) = 0.0
```

Comment:
Could you also do Atm(n)%physics_tendency_diag%t_dt(:) = 0.0? Being explicit about the bounds though is very fortranic and I'm fine with it!

Reply:
That's true, yeah.  Particularly since we allocate it in the line above, there's really no need to repeat the bounds (even for readability).